src/lib/insights/client.ts

```diff
@@ -0,0 +1,130 @@
+import {
```

Comment:
Internal imports should be after external imports btw. Maybe we can enforce that.

Reply:
I'm not very picky about it but there is a rule for it in eslint https://github.com/benmosher/eslint-plugin-import/blob/master/docs/rules/order.md